rust-toolchain.toml

```diff
@@ -1,5 +1,10 @@
 [toolchain]
 channel = "nightly-2022-09-15"
-targets = ["x86_64-unknown-linux-gnu", "x86_64-unknown-linux-musl", "x86_64-unknown-none", "wasm32-wasi"]
+targets = [
+    "aarch64-unknown-linux-musl",
```

Comment:
Why wasn't this necessary before this PR?

Reply:
Because it was specified within `flake.nix`, this file was not constructing a build toolchain, only the development one